pkg/config/validate.go

```diff
@@ -0,0 +1,87 @@
+package config
+
+import (
+	"fmt"
+	"io/ioutil"
+	"strings"
+
+	yaml "gopkg.in/yaml.v2"
+
+	yamlTojson "github.com/ghodss/yaml"
+	"github.com/skippbox/kompose/pkg/kobject"
+	"github.com/xeipuuv/gojsonschema"
+)
+
+func Validate(prefFile string) (kobject.ConvertOptions, error) {
+	var opt kobject.ConvertOptions
+
+	// Get the preference file name
+	prefFileContents, err := ioutil.ReadFile(prefFile)
+	if err != nil {
+		return opt, fmt.Errorf("Error Unmarshalling file - %s: %v", prefFile, err)
```

Comment:
@surajssd minor nitpick, this is not unmarshalling error, I think. Error message needs fixing.